src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -64,10 +64,8 @@ export const POPULARITY_WEIGHT: {
   rocketchat: 8,
 } as const;
 
-export const documentIntegrations: {
-  [key: string]: DocumentIntegration;
-} = {
-  fullstory: {
+export const documentIntegrationList: DocumentIntegration[] = [
```

Comment:
@NisanthanNanthakumar I needed to use an array so I decided to make the change of making the constant an array but create the object map below so we can get the best of both worlds :)